src/main/java/pl/touk/sputnik/processor/findbugs/FindBugsProcessor.java

```diff
@@ -30,17 +27,15 @@
 public class FindBugsProcessor implements ReviewProcessor {
     private static final String SOURCE_NAME = "FindBugs";
     private final CollectorBugReporter collectorBugReporter;
-    private final BuildDirLocator buildDirLocator;
 
     public FindBugsProcessor() {
         collectorBugReporter = createBugReporter();
-        buildDirLocator = BuildDirLocatorFactory.create();
     }
 
     @Nullable
     @Override
-    public ReviewResult process(@NotNull Review review) {
-        FindBugs2 findBugs = createFindBugs2(review);
+    public ReviewResult process(@NotNull Review review, @NotNull Configuration config) {
```

Comment:
All ReviewProcessors use this signature, but it's clunky. Maybe it would be more natural to inject configuration to a constructor.